README.md

```diff
@@ -1,47 +1,70 @@
-![Semantic UI Vue](/static/images/logo.png)
-
-[![Build Status](https://travis-ci.org/Semantic-UI-Vue/Semantic-UI-Vue.svg?branch=master)](https://travis-ci.org/Semantic-UI-Vue/Semantic-UI-Vue)
-[![codecov](https://codecov.io/gh/Semantic-UI-Vue/Semantic-UI-Vue/branch/master/graph/badge.svg)](https://codecov.io/gh/Semantic-UI-Vue/Semantic-UI-Vue)
-[![Join the chat at https://gitter.im/Semantic-UI-Vue/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Semantic-UI-Vue/Lobby)
-
-# Semantic UI Vue
-
-## Looking for maintainers!
+<p align="center">
+  <br /><img
+    width="192"
+    src="https://raw.githubusercontent.com/Semantic-UI-Vue/Semantic-UI-Vue/master/static/images/logo.png"
+    alt="Semantic UI Vue"
+  />
+
+</p>
+<p align="center">
+  <a href="https://travis-ci.org/Semantic-UI-Vue/Semantic-UI-Vue"
+    ><img src="https://travis-ci.org/Semantic-UI-Vue/Semantic-UI-Vue.svg?branch=master" alt="Pipeline status"
+       /></a>
+  <a href="https://crates.io/crates/starship"
+    ><img src="https://badge.fury.io/js/semantic-ui-vue.svg" alt="NPM version"
+  /></a>
+  <a href="https://codecov.io/gh/Semantic-UI-Vue/Semantic-UI-Vue"
+    ><img
+      src="https://codecov.io/gh/Semantic-UI-Vue/Semantic-UI-Vue/branch/master/graph/badge.svg"
+      alt="codecov"/></a
+  ><br />
+  <a href="https://gitter.im/Semantic-UI-Vue/Lobby"
+    ><img
+      src="https://badges.gitter.im/Join%20Chat.svg"
+      alt="Join the chat at https://gitter.im/Semantic-UI-Vue/Lobby"
+  /></a>
+</p>
+
+<h4 align="center">
+  <br />
+  <a href="https://semantic-ui-vue.github.io">Website</a>
+  ·
+  <a href="#-installation">Installation</a>
+  ·
+  <a href="#">Configuration (TBD)</a>
+</h4>
+
+---
```

Comment:
I didn't like this so much and I think we don't need it, the h1 itself divide the content